### DISCUSSION TITLE
Cannot run rabbitmq:3.8.16 image due to permissions errors

### DISCUSSION THREAD

**Discussion by WheresWardy** [docker-library/rabbitmq#518] [Q&A]
Since the rabbitmq:latest and rabbitmq:management images were pushed as a result of the 3.8.16 update, we're seeing permissions errors when trying to run those images in our CI environment:

```
$ uname -a
Linux ip-192-168-220-89 4.4.0-1092-aws #103-Ubuntu SMP Tue Aug 27 10:21:48 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

$ docker run -it rabbitmq:3.8.16
grep: /etc/rabbitmq/rabbitmq.conf: Permission denied
sed: can't read /etc/rabbitmq/rabbitmq.conf: Permission denied
```

Running the previous image is successful:

```
# docker run -it rabbitmq:3.8.15
Configuring logger redirection
2021-05-28 15:23:59.760 [debug] <0.288.0> Lager installed handler error_logger_lager_h into error_logger
...
```

We don't do anything special with these images other than run them, but noticed the base image for rabbitmq has been bumped to ubuntu:20.04 in this update, not sure if that's causing the issue somehow? (We can run the image ubuntu:20.04 perfectly OK for what it's worth).

**Comment by yosifkit** [comment]
I am unable to reproduce with the plain `docker run -it --rm rabbitmq:3.8.16`. What other docker flags are being used?

**Comment by OkenKhuman** [comment]
I also face the same issue. I followed [guide](docker run -it --rm --name rabbitmq -p 5672:5672 -p 15672:15672 rabbitmq:3-management) but results to `touch: setting times of '/etc/rabbitmq/rabbitmq.conf': Operation not permitted` for the latest version 3.8.16

For 3.8.15 it works perfectly

**Comment by WheresWardy** [comment]
> I am unable to reproduce with the plain `docker run -it --rm rabbitmq:3.8.16`. What other docker flags are being used?

No flags, the command I posted is literally the one used to run the image, and the total command output is included. I'll see if I can modify the entrypoint script to get it to tell me what's going on once it hits the file(s) it thinks it doesn't have permissions on.

**Comment by WheresWardy** [comment]
If it helps, we don't seem to have the issue with the Alpine version of the image, so might switch to running that instead.

**Comment by chanjarster** [comment]
I have the same issue, maybe related to docker version. On docker version 18.06.3-ce fail, version 18.09.0 pass.

**Comment by NL-TCH** [comment]
i have the same issue on my raspberrypi 4 it is returning the error:
`touch: setting times of '/etc/rabbitmq/rabbitmq.conf': Operation not permitted`
while using the following images:
with rabbitmq:3-management
with rabbitmq:3.8-management
with rabbitmq:3.8.16-management

it works while using the following image:
rabbitmq:3.8.15-management


**edit**
i was following the tutorial on the site https://www.rabbitmq.com/download.html
`docker run -it --rm --name rabbitmq -p 5672:5672 -p 15672:15672 rabbitmq:3-management`

**Comment by tianon** [comment]
@NL-TCH what you're missing is https://github.com/opencontainers/runc/pull/2750 -- you need a newer `runc` which is probably going to require Docker 20.10.4

**Comment by tianon** [comment]
... which actually might be at least related to the underlying issue for the OP here?  Can anyone reproduce with up-to-date Docker + runc?

**Comment by NL-TCH** [comment]
on the raspberry pi i already had docker version 20.10.7 (build f0df350)
i installed GO and Runc with the following tutorial: https://rolandsdev.blog/install-containerd-on-raspberry-pi/
but after restarting i still cannot install newer versions of rabbitmq on docker

**Comment by john-chrosniak** [comment]
@NL-TCH I had the same issue, fixed it by updating libseccomp2 on my pi 4
https://github.com/itzg/docker-minecraft-server/issues/755#issuecomment-781615497
